src/internal.c

```diff
@@ -6436,6 +6436,8 @@ int SendKexDhReply(WOLFSSH* ssh)
         return WS_BAD_ARGUMENT;
     }
 
+    memset(&sigKeyBlock, 0, sizeof sigKeyBlock);
```

Comment:
WMEMSET(..., sizeof(sigKeyBlock));

Mainly so it looks like everything else, more or less. And it'll use the memset wrapper.

Reply:
d'oh!

done.